code/modules/economy/supply_misc.dm

```diff
@@ -105,7 +143,25 @@ ABSTRACT_TYPE(/area/supply)
 			return FALSE
 	return ..()
 
-/obj/plasticflaps/ex_act(severity)
+// Slow down of flaps dependant on density of material
+// The more dense, the more slowdown
+/obj/strip_door/Crossed(atom/A)
+	..()
+	if (!src.flap_material)
+		return
+	if (isliving(A))
+		var/mob/living/M = A
+		var/density = src.flap_material.hasProperty("density") ? src.flap_material.getProperty("density") : 3
+		M.changeStatus("slowed", 2 SECONDS, density * 5)
```

Comment:
Is that a gut feeling, or have you tried it in practice? Because of how `/Crossed` gets applied, probably due to tick shenanigans, you actually only end up being slowed for a very brief period of time. In practice, it's not noticeable, especially not on lower density materials. I'd be happy to lower it.

The idea was just to incentivise experimenting with different materials for different effects.

Reply:
Gut feeling - I think that the lying-down slow is sufficient here